<ui version="4.0" >
 <class>InstantiationForm</class>
 <widget class="QDialog" name="InstantiationForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Instantiation Path</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="3" row="3" column="2" >
    <widget class="QListWidget" name="list" />
   </item>
   <item row="10" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>332</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0" colspan="3" >
    <widget class="QFrame" name="frame5" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item row="0" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>91</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" >
       <widget class="QPushButton" name="close_button" >
        <property name="text" >
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="3" row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>151</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2" >
    <widget class="QCheckBox" name="dbu_cb" >
     <property name="text" >
      <string>Coordinates in database units</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>:
Bottom</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Double-click to select the respective cell in the layout tree</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="title_lbl" >
     <property name="text" >
      <string>Instantiation path</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>332</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="QLineEdit" name="layout_le" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>37</width>
       <height>101</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Layout</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Top cell
:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="2" >
    <widget class="QCheckBox" name="abs_cb" >
     <property name="text" >
      <string>Absolute (accumulated) transformations</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>layout_le</tabstop>
  <tabstop>list</tabstop>
  <tabstop>dbu_cb</tabstop>
  <tabstop>abs_cb</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>InstantiationForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>298</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel" >
     <x>174</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
